README.md

```diff
@@ -95,20 +98,42 @@ This plugin decorates fastify with a `mail` object containing the following meth
 
 - `createMessage`: `function` - Generates a message from templates with context injected. 
   - Accepts the following arguments: 
-    - `recipients`: `array` - Array containing recipient[s] email address (`string`).
-    - `templates`: `string` - the relative path to the message's templates.
-    - `context`: `object` - Object containing context for the message.
+    - `message`: `array` -  Comma separated list or an array of recipients email addresses (`string`) that will appear on the To: field
+        - `from`: `string` - The email address the email is to be sent from.
+        - `to`: `array` - Comma separated list or an array of recipients email addresses (`string`) that will appear on the To: field
+        - `cc`: `array` - Comma separated list or an array of recipients email addresses (`string`) that will appear on the Cc: field
+        - `bcc`: `array` - Comma separated list or an array of recipients email addresses (`string`) that will appear on the Bcc: field
+        - `replyTo` : `string` - An email address that will appear on the Reply-To: field
+        - `subject`: `string` - The subject of the email with context injected.
+        - `html`: `string` - The HTML version of the message as an Unicode string, with context injected.
+        - `text` : `string` - The plaintext version of the message as an Unicode string, with context injected
+    - `templatePath`: `string` - the relative path to the message's templates.
```

Comment:
templatePath and context need to show they're inside an object like message above?

Reply:
they're not here. They're already destructured from sendmail (as they needed to be for a conditional).